lib/CFRPageActions.jsm

```diff
@@ -176,6 +176,25 @@ class PageAction {
     );
   }
 
+  /**
+   * Fetch the URL to the latest add-on xpi so the recommendation can download it.
+   * @param id          The add-on ID
+   * @return            A string for the URL that was fetched
+   */
+  async _fetchLatestAddonVersion(id) {
+    let url = null;
+    try {
+      const response = await fetch(`${ADDONS_API_URL}/${id}/`);
```

Comment:
Can you add `{credentials: omit}` just in case?

Reply:
Also, we need to file a follow-up to handle the case of this failing a little better than just showing nothing. For now , I think this is fine.